tests/foreman/destructive/test_leapp_satellite.py

```diff
@@ -0,0 +1,51 @@
+"""Tests for leapp upgrade of Satellite
+
+:CaseAutomation: Automated
+
+:CaseLevel: Component
+
+:CaseComponent: Upgrade
+
+:Assignee: lpramuk
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+import pytest
+from broker import Broker
+
+from robottelo.config import settings
+
+
+@pytest.mark.skipif(
+    str(settings.server.version.release).split('.')[1] != 11
+    and settings.server.version.rhel_version > 7,
+    reason='Run only on sat6.11el7',
+)
+def test_positive_leapp(target_sat):
+    """Upgrade satellite's RHEL version using leapp
+
+    :id: 761d4503-0b8c-494d-add5-b870fe1b90b9
+
+    :steps:
+        1. Get satellite on rhel7
+        2. Run job template that upgrades satellite to rhel8
+        3. Check results
+
+    :expectedresult:
+        1. Rhel version of the satellite has been updated
+
+    :parametrized: no
+    """
+    # Getting original RHEL version so we can increment it later in the test
+    orig_rhel_ver = target_sat.os_version.major
+    Broker().execute(job_template="satellite-leapp-upgrade", target_vm=target_sat.name)
```

Comment:
is this workflow/JT? can't find this in the tower repo

Reply:
see: 
https://infra-ansible-tower-01.infra.sat.rdu2.redhat.com/#/templates/job_template/233/details

Reply:
@Gauravtalreja1 See satlab-tower MR !665.